Gemfile.lock

```diff
@@ -0,0 +1,187 @@
+GIT
+  remote: https://github.com/aws-amplify/amplify-ci-support
+  revision: c1687ac33bc2d5420a29cd1dc4c5e90aeb36b701
+  branch: master
+  glob: src/fastlane/release_actions/*.gemspec
+  specs:
+    fastlane-plugin-release_actions (1.0.4)
+
+GEM
```

Comment:
Is this `Gemfile.lock` auto-generated? How was it generated, out of curiosity?

Reply:
I can give that a try. The only catch is that the `Fastfile` has to live under a `fastlane` subdirectory relative to where the `Gemfile` is (as far as I can tell). Maybe I can move the `Gemfile` under `.circleci` and move the `fastlane` folder there.? We'd end up with:
```
|[.circleci]
|-Gemfile
|-Gemfile.lock
|-[fastlane]
|-...fastlane stuff...
```

Reply:
I moved it to the `.circleci` subdir. We can always change it if that doesn't feel like the right place.